sdk/tables/azure-tables/review/tables.api.md

```diff
@@ -125,12 +134,8 @@ export interface GetEntityOptions extends coreHttp.OperationOptions {
 }
 
 // @public
-export type GetEntityResponse = TableQueryEntitiesWithPartitionAndRowKeyHeaders & TableEntityQueryResponse & {
-    _response: coreHttp.HttpResponse & {
-        bodyAsText: string;
-        parsedBody: TableEntityQueryResponse;
-        parsedHeaders: TableQueryEntitiesWithPartitionAndRowKeyHeaders;
-    };
+export type GetEntityResponse<T> = TableQueryEntitiesWithPartitionAndRowKeyResponse & {
+    value?: T;
```

Comment:
we have a TS guideline rule that says not to use `value` as a property name: https://azure.github.io/azure-sdk/typescript_design.html#general-dont-use-value

I'm not sure I fully understand the reason behind that rule as `value` isn't a reserved keyword, but I thought I'd mention it in case @bterlson has feelings/thoughts

Reply:
We'll be flattening `value` in the response object. I filed #10378 is tracking the work and will come right after this PR